views.go

```diff
@@ -502,6 +502,8 @@ func (lk *logKeeper) viewAllLogs(w http.ResponseWriter, r *http.Request) {
 		for line := range result.logLines {
 			_, err := w.Write([]byte(line.Data + "\n"))
 			if err != nil {
+				lk.logErrorf(r, "writing raw log lines from build '%s': %v", buildID, err)
```

Comment:
We should indicate the failure when it's not a "wrapped" message, per the style guide.

Reply:
Yeah I don't love this error message convention here, but I had to stop myself in the refactor cycle haha. We can revisit it in a subsequent ticket.